packages/react/src/components/Select/__tests__/Select-test.js

```diff
@@ -174,7 +174,6 @@ describe('Select', () => {
       render(
         <Select
           id="select-1"
-          label="Select label"
```

Comment:
This does not pass compilation, as there's no label attribute on Select.

Spent some time digging around and eventually concluded it should not be there, but happy to change this if someone else knows something I missed.

There's no explicitly defined label attribute. It does pass its ...other attributes on to the <select> element, but according to my searching (both looking at the Typescript definition of select and looking at docs for <select> online, I don't think it has a label attribute - it can be associated with a <label> element, but as far as I could tell setting a label= attribute is not a shorthand for an actual label element.

Reply:
I'll defer to @tay1orjones on this one 👀

Reply:
I believe it should be using the `labelText` prop, not `label`

Reply:
Great, thanks for weighing in! Updated to use `labelText` instead of `label`.